### PR TITLE
Add documentation about code-level-vulnerability in Python

### PR DESCRIPTION
**Pull request by juanjux**
### What does this PR do? What is the motivation?
It add information about the soon-to-be-public-beta code level vulnerability feature for Python.

- [X] Please merge after reviewing

### CODE REVIEW

**Comment by juanjux** [comment]
@DataDog/documentation ready to review and merge.

File: content/en/security/application_security/enabling/tracing_libraries/code_security/python.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: Enabling ASM for Python
+code_lang: python
+type: multi-code-lang
+code_lang_weight: 10
+aliases:
+  - /security_platform/application_security/getting_started/dotnet
+  - /security/application_security/getting_started/dotnet
```
Comment (gnufede) [inline content/en/security/application_security/enabling/tracing_libraries/code_security/python.md:8]:
```suggestion
  - /security_platform/application_security/getting_started/python
  - /security/application_security/getting_started/python
```

File: content/en/security/application_security/enabling/tracing_libraries/code_security/python.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+further_reading:
+    - link: "/security/application_security/add-user-info/"
+      tag: "Documentation"
+      text: "Adding user information to traces"
+    - link: 'https://github.com/DataDog/dd-trace-dotnet'
```
Comment (gnufede) [inline content/en/security/application_security/enabling/tracing_libraries/code_security/python.md:13]:
```suggestion
    - link: 'https://github.com/DataDog/dd-trace-py'
```

File: content/en/security/application_security/enabling/tracing_libraries/code_security/python.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      tag: "Source Code"
+      text: '.NET Datadog library source code'
```
Comment (gnufede) [inline content/en/security/application_security/enabling/tracing_libraries/code_security/python.md:15]:
```suggestion
      text: 'Python Datadog library source code'
```

File: content/en/security/application_security/enabling/tracing_libraries/code_security/python.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+You can monitor application security for Python apps running in Docker, Kubernetes, Amazon ECS, and AWS Fargate.
+
+{{% appsec-getstarted %}}
+
+## Enabling Code Security
+
+NOTE: Code-Level Vulnerability detection in Python is currently in beta.
+
+If your service runs a [tracing library version that supports Vulnerability Management for code-Level vulnerability detection][3], enable the capability by setting the `DD_IAST_ENABLED=true` environment variable and restarting your service.
```
Comment (gnufede) [inline content/en/security/application_security/enabling/tracing_libraries/code_security/python.md:32]:
```suggestion
If your service runs a [tracing library version that supports Vulnerability Management for Code-Level vulnerability detection][3], enable the capability by setting the `DD_IAST_ENABLED=true` environment variable and restarting your service.
```

File: content/en/security/application_security/enabling/tracing_libraries/code_security/python.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+To detect code-level vulnerabilities for your service:
```
Comment (gnufede) [inline content/en/security/application_security/enabling/tracing_libraries/code_security/python.md:34]:
```suggestion
To detect Code-Level vulnerabilities for your service:
```

**Review by gnufede — APPROVED** [review]

**Comment by buraizu** [comment]
Thanks, created DOCS-8411 to review

**Review by michaelcretzman — CHANGES_REQUESTED** [review]
this Python page was not integrated into the Enabling Code Security using Datadog Tracing Libraries page.

There should be a card for Python here, just like the other languages:
https://docs-staging.datadoghq.com/juanjux/code-level-vulns-python/security/application_security/enabling/tracing_libraries/code_security/

**Comment by juanjux** [comment]
> this Python page was not integrated into the Enabling Code Security using Datadog Tracing Libraries page.
> 
> There should be a card for Python here, just like the other languages: https://docs-staging.datadoghq.com/juanjux/code-level-vulns-python/security/application_security/enabling/tracing_libraries/code_security/

Card added, please take a look, thanks!

**Review by michaelcretzman — APPROVED** [review]

**Pull request merged**